godot-core/src/meta/mod.rs

```diff
@@ -55,6 +55,8 @@ mod traits;
 
 pub mod error;
 
+use std::{sync::OnceLock, thread::ThreadId};
```

Comment:
Maybe use fully qualified identifiers below (or `#[cfg]` this import, too), otherwise `wasm_nothreads` setup will get unused warnings.